projects/lyra/index.js

```diff
@@ -1,30 +1,38 @@
 const { getBlock } = require('../helper/getBlock')
 const { staking } = require('../helper/staking')
-const {sumTokens} = require('../helper/unwrapLPs')
+const { sumTokens } = require('../helper/unwrapLPs')
+
+const v1_0_Pools = ['0x7Af4e1cE484f40D927b9C90fB6905Df4376fc3F6', '0xd7d974E81382D05E8D9fc6d0d17d0d852e9806dd']
+const v1_1_LiquidityPool = ['0x2935CD347B79C319A6464fe3b1087170f142418C', '0x69B4B35504a8c1d6179fef7AdDCDB37A8c663BC9', '0x788843DE0Be1598155bFFaAB7Cfa2eCBd542E7f1']
```

Comment:
0x69B4B35504a8c1d6179fef7AdDCDB37A8c663BC9 appears twice on pools so liquidity there will be double counted